docker-compose.yaml

```diff
@@ -12,12 +12,18 @@ services:
       - ANTHROPIC_API_KEY=${ANTHROPIC_API_KEY}
       - OPENAI_API_KEY=${OPENAI_API_KEY}
       - GEE_SERVICE_ACCOUNT_PATH=${GEE_SERVICE_ACCOUNT_PATH}
+      - WATCHFILES_FORCE_POLLING=1
+      - DATABASE_URL=postgresql+psycopg2://postgres:postgres@db:5432/zeno-data
```

Comment:
Is this on the same PG instance as the langfuse tracing? How much separation are we thinking about on the db instance level?

Reply:
Yes - it's one the same PG instance, but in a separate database. My instinct is to say one PG instance for both but we can run that by Bitner to see if it's a bad idea in production (it definitely works for the demo)